client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -241,7 +241,8 @@ const Checkout = React.createClass( {
 				: '/checkout/thank-you/plans';
 		} else if ( selectedSite.options.is_domain_only && cartItems.hasDomainRegistration( cart ) && ! cartItems.hasPlan( cart ) ) {
 			const domainItem = find( cartItems.getAll( cart ), isDomainRegistration );
-			return domainManagementEdit( selectedSite.slug, domainItem.meta );
+
+			return domainManagementList( domainItem.meta );
```

Comment:
This approach won't work properly until the siteless checkout changes we have planned are merged, as the primary domain for the current site isn't set to the domain they are purchasing yet. 

What do you think about changing this to `selectedSite.slug` for now to fix this? We'll need to change this later anyway once there is no `selectedSite` so I think this is ideal for now.